src/main/fc/settings.yaml

```diff
@@ -18,6 +18,9 @@ tables:
   - name: pitot_hardware
     values: ["NONE", "AUTO", "MS4525", "ADC", "VIRTUAL", "FAKE"]
     enum: pitotSensor_e
+  - name: receiver_type
+    values: ["NONE", "PWM", "PPM", "SERIAL", "MSP", "SPI"]
+    enum: rxReceiverType_e
```

Comment:
Not that there's a problem with this. Just reminds me that I should get around to implementing enum validation in the settings generator.